src/js/angular/rest/autocomplete.rest.service.js

```diff
@@ -90,6 +92,24 @@ function AutocompleteRestService($http) {
         return $http.put(AUTOCOMPLETE_LABELS_ENDPOINT, label);
     }
 
+    function editLabelConfig(updatedLabel, originalLabel) {
```

Comment:
One more thing I'd ask you to do is to add an AutocompleteService and and use that in controllers instead of the rest client directly. For now you could expose only the add/edit label methods there where you can mode this logic for assembling the params below.